duktape.mk

```diff
@@ -0,0 +1,70 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS     += duktape
+DUKTAPE_VERSION := 2.6.0
+DEB_DUKTAPE_V   ?= $(DUKTAPE_VERSION)
+
+duktape-setup: setup
+	wget -q -nc -P $(BUILD_SOURCE) https://github.com/svaarala/duktape/releases/download/v$(DUKTAPE_VERSION)/duktape-$(DUKTAPE_VERSION).tar.xz
+	$(call EXTRACT_TAR,duktape-$(DUKTAPE_VERSION).tar.xz,duktape-$(DUKTAPE_VERSION),duktape)
+	$(SED) -i 's/gcc/cc/g' $(BUILD_WORK)/duktape/Makefile.cmdline
+	$(SED) -i 's|$$(CCLIBS)|$$(CCLIBS) $(LDFLAGS) $(CFLAGS)|g' $(BUILD_WORK)/duktape/Makefile.cmdline
+	mkdir -p $(BUILD_STAGE)/duktape/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/{lib/pkgconfig,include}
+	cp $(BUILD_MISC)/duktape/duktape.pc $(BUILD_STAGE)/duktape/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/lib/pkgconfig/duktape.pc
+	$(SED) -i 's|prefix=|prefix=$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)|g' \
+	$(BUILD_STAGE)/duktape/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/lib/pkgconfig/duktape.pc
+	$(SED) -i 's/Version:/Version: $(DUKTAPE_VERSION)/g' \
+	$(BUILD_STAGE)/duktape/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/lib/pkgconfig/duktape.pc
+
+
+ifneq ($(wildcard $(BUILD_WORK)/duktape/.build_complete),)
+duktape:
+	@echo "Using previously built duktape."
+else
+duktape: duktape-setup
+	cd $(BUILD_WORK)/duktape && python2 tools/configure.py --output-directory \
```

Comment:
You should change this to use python3 aka ``python$(PYTHON_MAJOR_V)``

Reply:
Just python3 would do fine

Reply:
I'll test python3 rn

Reply:
the configure file would need to be modified to work with python3

Reply:
Just tested python3 with this and errored